contracts/fee/FeeHandler.sol

```diff
@@ -60,26 +61,33 @@ contract FeeHandler is ReentrancyGuard, RoleModule, BasicMulticall {
     // @dev claim fees in feeToken from the specified markets
     // @param market the market from which to claim fees
     // @param feeToken the fee tokens to claim from the market
-    function claimFees(address market, address feeToken) external nonReentrant {
+    function claimFees(address market, address feeToken, uint256 version) external nonReentrant {
         if (_getBatchSize(feeToken) != 0) {
             revert Errors.InvalidFeeToken(feeToken);
         }
 
         uint256 feeAmount;
-        if (market == address(0)) {
+        if (version == v1) {
             feeAmount = IVaultGovV1(vaultV1.gov()).withdrawFees(address(vaultV1), feeToken, address(this));
-            _incrementAvailableFeeAmounts(1, feeToken, feeAmount);
-        } else {
+        } else if (version == v2) {
             feeAmount = FeeUtils.claimFees(dataStore, eventEmitter, market, feeToken, address(this));
```

Comment:
should validate that market is not address(0) here

Reply:
Ok, this has been added in the latest commit